packages/@atjson/document/src/object-annotation.ts

```diff
@@ -0,0 +1,7 @@
+import Annotation from './annotation';
+
+export default abstract class BlockAnnotation extends Annotation {
+  rank() {
+    return 1000;
```

Comment:
Thanks for explaining rank irl, and as you pointed out rank numbers should match :) https://github.com/CondeNast-Copilot/atjson/blob/ae08b75444fd091da016409edb0a9f871726f0a4/packages/@atjson/hir/src/hir-node.ts#L5-L13